### PR TITLE
19036 Added handling for Primary vs Holding + misc UI updates

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#19036

*Description of changes:*
- app version = 5.8.2
- added handling for PRIMARY vs HOLDING
- changed some "xxx Company" to "xxx Business" per uxpin changes
- added separate information blurb for horizontal amalg
- removed message box from review page (all amalgs) per uxpin changes
- added/updated unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

*Example:*
image

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-638-qk76i4dl.web.app

File: src/components/Amalgamation/BusinessTableSummary.vue [hunk 1]
```diff
@@ -103,10 +101,12 @@ export default class BusinessTableSummary extends Vue {
...
-    if (item.role === AmlRoles.AMALGAMATING) return 'Amalgamating Business'
-    if (item.role === AmlRoles.HOLDING && this.isAmalgamationFilingHorizontal) return 'Primary Company'
-    if (item.role === AmlRoles.HOLDING && this.isAmalgamationFilingVertical) return 'Holding Company'
-    return '(Unknown)' // should never happen
+    switch (item.role) {
+      case AmlRoles.AMALGAMATING: return 'Amalgamating Business'
+      case AmlRoles.HOLDING: return 'Holding Business'
+      case AmlRoles.PRIMARY: return 'Primary Business'
+      default: return '(Unknown)' // should never happen
+    }
```
Comment (PaulGarewal) [inline src/components/Amalgamation/BusinessTableSummary.vue:109]: Switch statements looking great again 👍

**Review by PaulGarewal — APPROVED** [review]
Looks good from what I can tell 👍

**Review by ketaki-deodhar — APPROVED** [review]
👍

**Pull request merged**